docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -0,0 +1,19 @@
+# SSPCommonTemplatesModificationReverted
```

Comment:
Could you please check https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md ? I think this will be similar to that runbook.